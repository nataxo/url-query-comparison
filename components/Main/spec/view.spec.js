import View from '../view';
import {shallow} from 'enzyme';

describe('main view', () => {
    it('renders', () => {
        const props = {
            equal: 'equal',
            difference: ['diff1', 'diff2'],
        };

        expect(shallow(<View {...props} />)).toMatchSnapshot();
    });
});
