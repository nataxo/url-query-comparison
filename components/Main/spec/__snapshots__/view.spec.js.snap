// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main view renders 1`] = `
<div>
  <main
    className="___main_16vyg_1"
  >
    <h1
      className="___h1_16vyg_1"
    >
      Compare Urls
    </h1>
    <Snippet>
      <_default />
    </Snippet>
    <Snippet>
      <h2
        className="___h2_16vyg_1"
      >
        Difference
      </h2>
      <_default
        titles={
          Array [
            "Param",
            "First Url",
            "Second Url",
          ]
        }
        values={
          Array [
            "diff1",
            "diff2",
          ]
        }
      />
    </Snippet>
    <Snippet>
      <h2
        className="___h2_16vyg_1"
      >
        Equal
      </h2>
      <_default
        titles={
          Array [
            "Param",
            "Value",
          ]
        }
        values="equal"
      />
    </Snippet>
  </main>
  <Footer />
</div>
`;
